src/wp-includes/cache.php

```diff
@@ -102,6 +102,62 @@ function wp_cache_flush() {
 
 	return $wp_object_cache->flush();
 }
+/**
+ * Removes all cache items in a group.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool True on success, false on failure group not found.
+ */
+function wp_cache_flush_group( $group ) {
+
+	global $wp_object_cache;
+
+	// if group is an array loop and call each key in the array
+	if ( is_array( $group ) ) {
+		array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+		return true;
+	}
+
+	if ( method_exists( $wp_object_cache, 'flush_group' ) ) {
+		return $wp_object_cache->flush_group( $group );
+	}
+
+	return false;
+}
+
+
+/**
+ * stores the mapping of linked cache groups
+ *
+ * @since 6.0.0
+ *
+ * @return array keyed by group, value is array of linked groups
+ */
+function wp_cache_get_linked_meta() {
+
+	/**
+	 * some the cache groups are linked, so we need to flush them both if one is called
+	 * this is the mapping for these groups
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param array array of linked cache groups that need to delete as a set.
+	 */
+	return apply_filters(
+		'wp_cache_get_linked_meta',
+		array(
+			'users'     => array( 'user_meta' ),
+			'user_meta' => array( 'users' ),
+		)
+	);
+}
```

Comment:
This function is not needed for this patch right?

Reply:
yes I will remove it